activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -62,7 +62,8 @@ class SQLite3Adapter < AbstractAdapter
         time:         { name: "time" },
         date:         { name: "date" },
         binary:       { name: "blob" },
-        boolean:      { name: "boolean" }
+        boolean:      { name: "boolean" },
+        bigint:       { name: "bigint" },
```

Comment:
My guess is that we can probably remove most of the entries in the `Sqlite3Adapter` but let's explore that in another PR. As long as they don't make a transformation or have some non standard options, they won't affect behavior.

In terms of consistency. In my opinion we should target removing instead of adding new entries. Especially in sqlite3 where the column types don't really matter.